apps/etterlatte-saksbehandling-ui/client/src/components/person/behandlingsslistemappere.ts

```diff
@@ -19,6 +19,10 @@ export function mapAarsak(aarsak: BehandlingOgRevurderingsAarsakerType) {
       return 'Søskenjustering'
     case Revurderingaarsak.UTLAND:
       return 'Ut-/innflytting til Norge'
+    case Revurderingaarsak.EKSPORT:
```

Comment:
Hvordan fikk du ikke feil sist? :D

Reply:
Det er to forskjellige lister. Dette gjelder listevisningen for behandlinger, der hadde jeg ingen behandlinger ennå så la ikke merke til at det var enda en mapping for dette 😛